specs/provider.yaml

```diff
@@ -1043,3 +1080,33 @@ definitions:
     additionalProperties: false
     required:
     - message
+
+  ResourceMeasures:
+    type: object
+    description: |
+      Wrapper object for resource measures body.
+    properties:
+      body:
+        $ref: '#/definitions/ResourceMeasuresBody'
+    additionalProperties: false
+    required:
+      - body
+
+  ResourceMeasuresBody:
+    type: object
+    description: |
+      Object used to hold information about a resource and its features usage.
```

Comment:
If you wanted to reword:
"Object describing a resource's feature usage for a billing period."